### PR TITLE
ENG-962 Instructions on getting set up with gitpod.io

### PR DESCRIPTION
**Pull request by agbasioscar**
Decide on workflow (Forking?) How do we view their gitpod environment?

Deliverables:

* Documentation on how to get coding/begin the evaluation on gitpod.io

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +56,23 @@ Add props to the `ImageCarousel` component that controls the `showThumbs` and `s
 
 In your use of the `ImageCarousel` component for Task 1, make sure the thumbnails are not visible, but the indicators are visible.
 
-### 3. Style the `ImageCarousel` component
-Using the mockup included in the root level of the project, position and size the `ImageCarousel` component on the page, with considerations for how responsive it is on smaller screen sizes.
-
 ### 4. Add a loading indicator
 While waiting for the results from the image API to be returned and rendered, display a loading indicator. A spinner image can be found under the `public` directory. The spinner should be displayed at 100% of its size in the middle of the page.
```
Comment (jrisebor) [inline README.md]: tasks 2 & 4 here also seem like frontend tasks (I guess 4 is a bit of both), should they be under the frontend heading? Also I think the numbering needs to be corrected as well.

File: README.md [hunk 2]
```diff
@@ -33,7 +33,19 @@ Once the application's dependencies are installed, the start script will execute
 
 You're now ready to begin your tasks!
 
-## Tasks
+## How to Share Workspaces
+Sharing running workspaces makes it possible to quickly look at a workspace together with a (remote) colleague [Gitpod Sharing Workspace](https://www.gitpod.io/docs/sharing-and-collaboration#sharing-running-workspaces).
+To share your workspace navigate to the workspaces page:
+
+#### 1. Move your mouse over the workspace you want to share (change the filter to All if you don’t see your workspace.
```
Comment (jrisebor) [inline README.md]: Should this be H3 instead of H4? (i.e. `###` instead of `####`?)

**Comment by guil-uberflip** [comment]
Approved

File: README.md [hunk 3]
```diff
@@ -44,11 +56,23 @@ Add props to the `ImageCarousel` component that controls the `showThumbs` and `s
...
 
+## Frontend Task
+### 1. Style the `ImageCarousel` component
+Using the mockup included in the root level of the project, position and size the `ImageCarousel` component on the page, with considerations for how responsive it is on smaller screen sizes.
+
+### 2. Add Navigation bar
+Make the navigation bar look similar to the mockup.jpg
+
+### 4. Response name and description 
+Make sure the names and description of the exact instead of {name} or {description}
```
Comment (robmontebelli) [inline README.md]: Is there a typo here? Also, I am not sure what the response and description are 🤔

File: README.md [hunk 4]
```diff
@@ -44,11 +56,23 @@ Add props to the `ImageCarousel` component that controls the `showThumbs` and `s
...
+
+### 5. Toggle functionality
```
Comment (robmontebelli) [inline README.md]: Which toggle is it referring to?

**Review by robmontebelli — COMMENTED** [review]
We now have 2 mockup files (one png that's 11 MB and should be smaller) and I'm not sure what the png file refers to

**Review by jrisebor — CHANGES_REQUESTED** [review]

**Pull request merged**